packages/web-client/app/templates/card-pay/balances.hbs

```diff
@@ -8,8 +8,9 @@
   </Boxel::Button>
 </section>
 
-{{#if this.isShowingDepositWorkflow}}
-  <Boxel::Modal @onClose={{set this "isShowingDepositWorkflow" false}}>
-    <CardPay::DepositWorkflow />
-  </Boxel::Modal>
-{{/if}}
\ No newline at end of file
+<Boxel::Modal
```

Comment:
I think we should check with design if they would prefer closing on Escape or not, but we can leave that for when the design team starts playing with the UI since this seems like a fairly straightforward change. Will add this to CS-627.

Reply:
I think the escape key for modal is standard for accessibility. (Focus trap is also standard, but we haven't implemented it yet as you mentioned.) https://www.w3.org/TR/wai-aria-practices/#dialog_modal